x-pack/legacy/plugins/siem/server/kibana.index.ts

```diff
@@ -27,6 +29,16 @@ const APP_ID = 'siem';
 export const amMocking = (): boolean => process.env.INGEST_MOCKS === 'true';
 
 export const initServerWithKibana = (kbnServer: Server) => {
+  if (kbnServer.plugins.alerting != null) {
+    const type = signalsAlertType({
+      logger: kbnServer.newPlatform.coreContext.logger.get('plugins', APP_ID),
+    });
+    if (isAlertExecutor(type)) {
+      kbnServer.plugins.alerting.setup.registerType(type);
+    }
+  }
+  kbnServer.injectUiAppVars('siem', async () => kbnServer.getInjectedUiAppVars('kibana'));
+
   // bind is so "this" binds correctly to the logger since hapi server does not auto-bind its methods
   const logger = createLogger(kbnServer.log.bind(kbnServer));
```

Comment:
Can we replace this logger with the module in NP, or do we need two distinct instances here?

Reply:
I'm moving forward on the assumption that we can; please correct me if I'm wrong.

Reply:
Yes, I will take the same path that you did.